exercises/practice/clock/.docs/instructions.append.md

```diff
@@ -0,0 +1,56 @@
+# Instructions append
+
+## Representing your class
+
+When working with and debugging objects, it's important to have a string representation of that object. For example, if you were to create a `datetime.datetime` object in a REPL environment, you could see its string representation:
+
+```python
+>>> from datetime import datetime
+>>> datetime(2022, 5, 4)
+datetime.datetime(2022, 5, 4, 0, 0)
+```
+
+In this excersize, you will create a custom object, `Clock`, that handles times without dates. To do this in python, you would create a `class` to define its data and behavior.
+
+One aspect of a class' behavior is this string prepresentation. By default, a class' string representation is not very helpful:
+
+```python
+>>> Clock(12, 34)
+<Clock object st 0x... >
+```
+
+To create a more helpful representation, you will need to define a `__repr__` method on your class that returns a string.
+
+According to the [specification for a `__repr__` method](https://docs.python.org/3/reference/datamodel.html#object.__repr__), you should ideally have the method return a string that could be used to recreate the object, if you were to copy and paste it into a new environment. 
+
+For example, a `Clock` that represents 11:30 AM could look like this: `Clock(11, 30)`.
+
+
+Defining a `__repr__` method is a great thing to do for every class you create. When you do, you should consider:
+
+- The information you're returning from this method will be beneficial when you're debugging issues.
+
+- Ideally, you should return a string that a user could copy, paste into a python environment, and have a copy of the object.
+
+- If that's not practical, return a description between angle brackets, as in `< ...a practical description... >`.
+
+
+### String conversion
+
+As useful as the `__repr__` method is, sometimes you'll want an alternative string representation. For example, let's look at the `datetime.datetime` class again:
```

Comment:
```suggestion
In addition to the `__repr__` method, sometimes you'll want an alternative string representation.
This might be used to format the object for program output.
For example, let's look at the `datetime.datetime` class again:
```